analyzer/codechecker_analyzer/buildlog/build_manager.py

```diff
@@ -123,6 +115,19 @@ def perform_build_command(logfile, command, context, keep_link, silent=False,
             LOG.error("Intercept-build is required to run CodeChecker in "
                       "OS X.")
             sys.exit(1)
+    # Otherwise try to fall back to intercept-build.
```

Comment:
Now the logger detection looks like this:
```py
if ld_logger:
  if LINUX:
    ...
  elif WINDOWS:
    ...
  else:
    ...
elif intercept_build:
  ...
else:
  ...
```

Wouldn't it be better to use the following check:
```py
if ld_logger:
  ...
elif intercept_build:
  ...
else: # No intercept build or ld-logger
  if LINUX:
    ...  # Install ld-logger or intercept-build
  elif WINDOWS:
    ...  # Original log message.
  else:
    ...  # Install intercept-build
```